### ISSUE TITLE
Shifter Zerus does not show transformations

### ISSUE THREAD

**Issue by HugoKornelis** [HearthSim/Hearthstone-Deck-Tracker#2489]
I have Shifter Zerus in my current arena run. I had him in my opening hand, kept him (her? it?) there for a few turns, and then played him.
In the replay, the various forms he had while in my hand are not shown. This makes it hard to review the game and the choices I made. Also, after playing the card it appears as Shifter Zerus on the board, which makes the rest of the game hard to follow. (After being hit, it appears to suvive with negative health; then hits a 2-health minion with its 1 attack and kills it).

Would it be possible to show the card in its current form in the replay? (Ideally including some marker to recognize it as "created by Shifter Zerus")

**Comment by azeier** [comment]
Local replay support has now been fully removed.
